packages/server/tests/utils.ts

```diff
@@ -319,4 +321,9 @@ export default class TestUtils {
         return {contractExecuteTimestamp, contractExecutedTransactionId};
     };
 
+    signRawTransaction = async (tx: TransactionRequest, privateKey) => {
+        const provider = new ethers.providers.JsonRpcProvider('http://localhost:7546');
```

Comment:
I think that we should define the provider in the `describe` within the acceptance test file and pass it as an argument here. We have the localhost:7546 defined there